src/executive.rs

```diff
@@ -319,7 +323,7 @@ fn create<B: DB + 'static>(
         &address,
         balance,
         // The init nonce for a new contract is one, see above documents.
-        U256::one(),
+        U256::zero(),
```

Comment:
Since we change this value from `1 => 0`, please remove the comment in previous line.